android/sdl_android/src/androidTest/java/com/smartdevicelink/test/VehicleDataHelper.java

```diff
@@ -5,16 +5,19 @@
 import com.smartdevicelink.proxy.rpc.BodyInformation;
 import com.smartdevicelink.proxy.rpc.ClusterModeStatus;
 import com.smartdevicelink.proxy.rpc.DeviceStatus;
+import com.smartdevicelink.proxy.rpc.DoorStatus;
```

Comment:
@kostyaBoss You can revert the changes in `VehicleDataHelper.java`.

You can delete the unused import on line 8.